run.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+include 'vendor/autoload.php';
+include 'funcs_scripts.php';
+include 'funcs_utils.php';
+
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputOption;
+
+// consts
+const CURRENT_CMS_MAJOR = '5';
+const BRANCH_OPTIONS = ['next-minor', 'next-patch'];
+const DEFAULT_BRANCH = 'next-minor';
+const DEFAULT_ACCOUNT = 'creative-commoners';
+const DATA_DIR = '_data';
+const MODULES_DIR = '_modules';
+const TOOL_URL = 'https://github.com/silverstripe/module-standardiser';
+const PR_TITLE = 'MNT Run module-standardiser';
+const PR_DESCRIPTION = 'This pull-request was created automatically by [module-standardiser](' . TOOL_URL . ')';
+
+// global variables
+$MODULE_DIR = '';
+$PRS_CREATED = [];
+$REPOS_WITH_PRS_CREATED = [];
+$OUT = null;
+
+$app = new Application();
+$app->register('update')
+    ->setDescription('The main script of module-standardiser')
+    ->addOption(
+        'cms-major',
+        null,
+        InputOption::VALUE_REQUIRED,
+        'The CMS major version to use (default: '. CURRENT_CMS_MAJOR .')'
+    )
+    ->addOption(
+        'branch',
+        null,
+        InputOption::VALUE_REQUIRED,
+        'The branch type to use - ' . implode('|', BRANCH_OPTIONS) . ' (default: ' . DEFAULT_BRANCH . ')'
+    )
+    ->addOption(
+        'only',
+        null,
+        InputOption::VALUE_REQUIRED,
+        'Only include the specified modules (without account prefix) separated by commas '
+        . 'e.g. silverstripe-config,silverstripe-assets'
+    )
+    ->addOption(
+        'exclude',
+        null,
+        InputOption::VALUE_REQUIRED,
+        'Exclude the specified modules (without account prefix) separated by commas '
+        . 'e.g. silverstripe-mfa,silverstripe-totp'
+    )
+    ->addOption(
+        'dry-run',
+        null,
+        InputOption::VALUE_NONE,
+        'Do not push to github or create pull-requests'
+    )
+    ->addOption(
+        'account',
+        null,
+        InputOption::VALUE_REQUIRED,
+        'GitHub account to use for creating pull-requests (default: ' . DEFAULT_ACCOUNT . ')'
+    )
+    ->addOption(
+        'no-delete',
+        null,
+        InputOption::VALUE_NONE,
+        'Do not delete _data and _modules directories before running'
+    )
+    ->setCode(function (InputInterface $input, OutputInterface $output): int {
+        // This is the code that is executed when running the 'update' command
+
+        // variables
+        global $MODULE_DIR, $OUT, $PRS_CREATED, $REPOS_WITH_PRS_CREATED;
+        $OUT = $output;
+
+        // validate system is ready
+        validate_system();
```

Comment:
Could we check also that `git` command is installed in the system? Since user can clone it through the desktop terminal and run your command in Docker container where git is not installed.

Reply:
How would they even clone this repo without having `git` installed :laughing:

Reply:
> Since user can clone it through the desktop terminal and run your command in Docker container where git is not installed.

Probably, it's an edge case, but anyway I think we should check that command exists

Reply:
Updated